resources/views/faults/list.blade.php

```diff
@@ -0,0 +1,98 @@
+<div id="faults-table"></div>
+<script src="js/moment.min.js"></script>
+<script type="application/javascript">
+    $(document).ready(function () {
+        function status_translate(cell) {
+            if (typeof cell !== "string") {
+                cell = cell.getValue();
+            }
+            var translations = {
+                UNSEEN: "@lang('faults.unseen')",
+                SEEN: "@lang('faults.seen')",
+                DONE: "@lang('faults.done')",
+                WONT_FIX: "@lang('faults.wont_fix')"
+            };
+            var colors = {
+                UNSEEN: "text-secondary",
+                SEEN: "text-info",
+                DONE: "text-success",
+                WONT_FIX: "text-danger"
+            };
+            return '<div class="font-italic ' + colors[cell] + ' text-right">' + translations[cell] + '</div>';
+        }
+
+        var button = function (cell, formatterParams) {
+            if (formatterParams['status'] === "DONE") {
+                var style = "btn-success";
+                var text = "@lang('faults.done')";
+            } else if (formatterParams['status'] === "WONT_FIX") {
+                var style = "btn-danger";
+                var text = "@lang('faults.wont_fix')";
+            } else if (formatterParams['status'] === "UNSEEN") {
+                var style = "btn-danger";
+                var text = "@lang('faults.reopen')";
+            }
```

Comment:
Shouldn't we use `switch` instead?

Reply:
That probably looks better, have changed that.